src/Model/Behavior/SearchBehavior.php

```diff
@@ -22,7 +22,11 @@ class SearchBehavior extends Behavior
     /**
      * Default config for the behavior.
      *
+     * You can overwrite default empty values using emptyValues key
+     * when initialise the behavior
```

Comment:
initializating

Reply:
Proper English is very posh 💛 :trollface:

Reply:
Sry :) Typo fixed.

Reply:
lol. done